src/network/message.rs

```diff
@@ -38,6 +38,10 @@ use crate::util::merkleblock::MerkleBlock;
 /// This limit is not currently enforced by this implementation.
 pub const MAX_INV_SIZE: usize = 50_000;
 
+/// Maximum size, in bytes, of a an encoded message
```

Comment:
Woops, sorry I missed this on the first review.
```suggestion
/// Maximum size, in bytes, of an encoded message
```